src/styles/main.scss

```
@@ -11,4 +11,37 @@ body {
 }
 
 @import "_header.scss";
-@import "_catalog.scss";
+// @import "_catalog.scss";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
+  justify-content: center;
```

```suggestion
.catalog {
  display: grid;
  grid-template-columns: repeat(4, 200px);
  gap: 48px;
  justify-content: center;
  
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, 200px);
  }
  @media (min-width: 768px) {
    grid-template-columns: repeat(3, 200px);
  }
  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, 200px);
  }
```

try to add media like this

---

fixed